installer/templates/new/web/views/error_view.ex

```diff
@@ -13,5 +13,17 @@ defmodule <%= application_module %>.ErrorView do
   # template is found, let's render it as 500
   def template_not_found(_template, assigns) do
     render "500.html", assigns
+  end<% else %>def render("404.json", _assigns) do
+    %{"errors" => %{"detail" => "Page not found"}}
   end
+
+  def render("500.json", _assigns) do
+    %{"errors" => %{"detail" => "Server internal error"}}
```

Comment:
Ok! @stevedomin sorry for one extra back and forth but can you please change them to use atom keys and fix the generated tests accordingly? Thank you!